devdocs/process.md

```diff
@@ -118,7 +122,7 @@ This workflow is an adaptation of the [GitHub flow](https://guides.github.com/in
    * Push your branch to the committer repo (push to the fork if you do not 
      have push permission to the committer repo), if you haven't done that already.
    
-   * Create a pull request (PR). For the pull request name, copy paste the relevant
+   * <a name="creating-a-pull-request"></a>Create a pull request (PR). For the pull request name, copy paste the relevant
      issue name.<br>
```

Comment:
"... name *and include the issue number as well*". It's not uncommon that the issue number is left off.